packages/blade-mcp/src/tools/publishLinesOfCodeMetric.ts

```diff
@@ -77,37 +91,87 @@ const publishLinesOfCodeMetricToolSchema = {
     )
     .nonempty()
     .describe('Breakdown of line changes per file in this operation.'),
-  linesOfCodeAddedTotal: z
-    .number()
-    .int()
-    .nonnegative()
-    .describe('Total lines added across all files.'),
-  linesOfCodeRemovedTotal: z
+  linesAddedTotal: z.number().int().nonnegative().describe('Total lines added across all files.'),
+  linesRemovedTotal: z
```

Comment:
do we want to move out of this "line" terminology into something like "codeDiffAdditions", "codeDiffSubtraction" or "changeAdditions", "changeSubtractions"?